.github/workflows/linters.yml

```diff
@@ -0,0 +1,22 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
+  rubocop:
+    name: Rubocop
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-ruby@v1
+        with:
+          ruby-version: 3.0.x
+      - name: Setup Rubocop
+        run: |
+          gem install --no-document rubocop -v '>= 1.0, < 2.0' # https://docs.rubocop.org/en/stable/installation/
```

Comment:
Hello @misheck12, you did a wonderful work actually, I really appreciate it. Unfortunately I can not proceed by approving the work as the linters are falling. Kindly check them out and push again. I hope you will make it 👌
Happy coding 😺

Reply:
sure

Reply:
hello @keza681 i have fixed the errors please review this

Reply:
Hi @misheck12, well done. Can you merge it now?